### DISCUSSION TITLE
Installation Error

### DISCUSSION THREAD

**Discussion by BrokenTippedQuill** [leokhoa/laragon#681] [Q&A]
I installed Laragon on my desktop and it worked just fine but when I went to my laptop and do it there it shows me an error and I don't know what to do, please help.

<img width="393" alt="Screenshot 2024-02-07 123132" src="https://github.com/leokhoa/laragon/assets/159223517/9333c81d-1338-4f95-85fc-bca2bc2f86f9">

<img width="419" alt="Screenshot 2024-02-07 123146" src="https://github.com/leokhoa/laragon/assets/159223517/f940b4e1-0804-4396-8fb4-d9a4f60a2080">

**Comment by Pen-y-Fan** [comment]
Both errors are related to permission.

1. Try a re-install as an administrator or privileged account
2. Check the permission, for your account, see [How to set up Laragon on a new Windows computer (part 2) - permissions](https://pen-y-fan.github.io/2022/11/08/how-to-set-up-laragon-on-a-new-windows-computer-part-2/)
